src/sql/parts/profiler/editor/profilerEditor.ts

```diff
@@ -199,8 +200,16 @@ export class ProfilerEditor extends BaseEditor {
 		}));
 		let dropdownContainer = document.createElement('div');
 		dropdownContainer.style.width = '150px';
+		dropdownContainer.style.paddingRight = '5px';
 		this._viewTemplateSelector.render(dropdownContainer);
 
+		this._connectionInfoText = document.createElement('div');
+		this._connectionInfoText.style.paddingRight = '5px';
+		this._connectionInfoText.innerText = '';
+		this._connectionInfoText.style.textAlign = 'center';
+		this._connectionInfoText.style.display = 'flex';
+		this._connectionInfoText.style.alignItems = 'center';
```

Comment:
Better to use css files for all this styling unless there's some reason it needs to be done here